letsencrypt/cli.py

```diff
@@ -315,7 +322,8 @@ def _handle_subset_cert_request(config, domains, cert):
              ", ".join(domains),
              br=os.linesep)
     if config.expand or config.renew_by_default or zope.component.getUtility(
-            interfaces.IDisplay).yesno(question, "Expand", "Cancel"):
+            interfaces.IDisplay).yesno(question, "Expand", "Cancel",
+                                       default=True):
```

Comment:
Talked about this out of band and I believe we agreed we should error out. Ping me if you reached a different conclusion.